lib/scholar/metrics/similarity.ex

```diff
@@ -0,0 +1,61 @@
+defmodule Scholar.Metrics.Similarity do
+  @moduledoc """
+  Similarity metrics between 1-D tensors.
+  """
+
+  import Nx.Defn
+  import Scholar.Shared
+
+  @doc """
+  The Jaccard similarity (also known as Jaccard similarity coefficient, or Jaccard index)
+  is a statistic used to measure similarities between two sets. Mathematically, the calculation
+  of Jaccard similarity is the ratio of set intersection over set union.
+
+  J(A, B) = |A∩B| / |A∪B|
```

Comment:
You can write this between a pair of `$$` and use the LaTeX syntax :)

Reply:
done